src/odb/src/db/dbNet.cpp

```diff
@@ -727,6 +727,30 @@ bool dbNet::rename(const char* name)
   return true;
 }
 
+void dbNet::swapNetNames(dbNet* source)
```

Comment:
Does this need to be undo-able?

Reply:
No. But I like your thinking !

(The reason I can _boldy_ claim no is that this one relates to the output buffers connected to top level pins -- those top level connections are always called by the top level port name, even if wrongly named).

Reply:
Added undo capability for this method.